includes/gateway/abstract-omise-payment-base-card.php

```diff
@@ -63,19 +63,23 @@ private function prepareChargeData($order_id, $order, $omise_customer_id, $card_
 		$data = [
 			'amount' => Omise_Money::to_subunit($order->get_total(), $currency),
 			'currency' => $currency,
-			'description' => apply_filters(
-				'omise_charge_params_description',
-				'WooCommerce Order id ' . $order_id,
-				$order
-			),
-			'return_uri' => $this->getRedirectUrl('omise_callback', $order_id, $order),
-			'metadata' => $this->getMetadata(
+			'description' => 'WooCommerce Order id ' . $order_id,
+			'return_uri' => $this->get_redirect_url('omise_callback', $order_id, $order),
+			'metadata' => $this->get_metadata(
```

Comment:
Got error because `get_redirect_url` and `get_metadata` do not exist.

Reply:
Oops. Accidentally removed the trait. Sorry for that.